### ISSUE TITLE
Sleeping State Suggestions

### ISSUE THREAD

**Issue by adamsimecka** [arcbtc/LNURLPoS#22]
The device does not appear to automatically go into PretendSleeping if the device is in a deep sleep mode. This means that if the device is sleeping and it is then plugged in, the device does not charge (according to Blackcofee). If it is possible, I believe we should trigger PretendSleeping as soon as a charging voltage is detected - although that might be a challenge since the device is in deep sleep and may not be able to execute those instructions from that state.

Similarly, the device does not appear to go into a normal deep sleep automatically from a PretendSleeping state. Meaning, if I am charging the device and it is in the Pretend Sleeping state, but then I unplug it without waking it up, it seems to remain in the PretendSleeping state which likely will cause severe battery drain.

While in normal deep sleep all keys in column 1 will wake the device. While in PretendSleeping state, only the reset (*) key will wake the device. This can be confusing for the user if there are different waking rules depending on the state.

**Comment by adamsimecka** [comment]
A suggestion to the issues above, would be to map one of the physical buttons to force it into a deep sleep like the way it is done on the LilyGo T-display factory test example. I don't know if that would solve the problem of the battery supposedly not charging while in deep sleep.

**Comment by arcbtc** [comment]
Aye, it doesnt need to pretend sleep, we should just remove that and have it actually sleep

**Comment by adamsimecka** [comment]
Yes. I completely agree. The original issue was during a time that Blackcoffeexbt was having issues with charging while sleeping. I believe those have been resolved. So, we should not need pretend sleeping any longer.
